fuzz/fuzz_targets/markdown.rs

```diff
@@ -9,7 +9,15 @@ fuzz_target!(|data: &[u8]| {
             s,
             &markdown::Options::gfm()
         );
+        let _ = markdown::to_html_with_options(
+            s,
+            &markdown::Options {
+                parse: markdown::ParseOptions::mdx(),
+                ..Default::default()
+            }
+        ).ok();
         let _ = markdown::to_mdast(s, &markdown::ParseOptions::default());
         let _ = markdown::to_mdast(s, &markdown::ParseOptions::gfm());
+        let _ = markdown::to_mdast(s, &markdown::ParseOptions::mdx()).ok();
```

Comment:
What does the `.ok()` add? I believe it’ll work fine without it: that it’s a result and doesn’t panic?

Reply:
My impression is/was that some errors are to be expected with MDX?
My goal is to capture/ignore known errors, and only look for unexpected `panic!`.

That understand could be incorrect though.
Is the goal with the `mdx` parser to never throw?

---

in the meantime removed `.ok`

Reply:
also related https://github.com/wooorm/markdown-rs/pull/26#issuecomment-1296280425